### PR TITLE
Angular 16 support

### PR DESCRIPTION
**Pull request by jornare**
icons-angular updated to version 16.
Builds with Angular 16, and with updated dependencies.

### CODE REVIEW

**Comment by Nicoss54** [comment]
Hi now with angular 16, we can pass some meta to say that the input is required or not, I think it could be a good idea to use this feature in the directive to make the input type required.

What do you think of that ?

**Comment by jornare** [comment]
> Hi now with angular 16, we can pass some meta to say that the input is required or not, I think it could be a good idea to use this feature in the directive to make the input type required.
> 
> What do you think of that ?

Absolutely. However, I would really like to see the 16 support asap, then extend with this feature in a separate pr. I guess it could cause breaking changes for some..? There could also be other refactoring issues we could update here (Skip module?), but I don't have the time nor insight at the moment to take that into consideration.

**Comment by Nicoss54** [comment]
> > Hi now with angular 16, we can pass some meta to say that the input is required or not, I think it could be a good idea to use this feature in the directive to make the input type required.
> > What do you think of that ?
> 
> Absolutely. However, I would really like to see the 16 support asap, then extend with this feature in a separate pr.

Okkey thanks for the work and answer :)

File: packages/icons-angular/angular.json [hunk 1]
```diff
@@ -143,8 +143,6 @@
   "cli": {
     "analytics": false,
     "schematicCollections": [
-      "@angular-eslint/schematics",
-      "@angular-eslint/schematics"
-    ]
+      "@angular-eslint/schematics"    ]
```
Comment (wzhudev) [inline packages/icons-angular/angular.json]: Please format this. :)

**Review by wzhudev — APPROVED** [review]
Overall, LGTM. Thanks!

File: packages/icons-angular/angular.json (hunk 1 above)
Comment (jornare) [reply packages/icons-angular/angular.json]: done

File: packages/icons-angular/package.json [hunk 2]
```diff
@@ -16,52 +16,52 @@
...
-    "@angular/compiler": "^15.0.1",
-    "@angular/compiler-cli": "^15.0.1",
-    "@angular/forms": "^15.0.1",
-    "@angular/language-service": "^15.0.1",
-    "@angular/platform-browser": "^15.0.1",
-    "@angular/platform-browser-dynamic": "^15.0.1",
-    "@angular/router": "^15.0.1",
+    "@angular-devkit/build-angular": "^16.0.0",
+    "@angular-eslint/builder": "^16.0.0",
+    "@angular-eslint/eslint-plugin": "^6.0.0",
```
Comment (wzhudev) [inline packages/icons-angular/package.json]: This should be `^16.0.0`. Please make sure that CI passes.

**Review by wzhudev — CHANGES_REQUESTED** [review]

File: packages/icons-angular/package.json (hunk 2 above)
Comment (jornare) [reply packages/icons-angular/package.json]: well spotted, I'll fix right away

**Comment by jornare** [comment]
Seems like the CI needs a docker/node version update.

**Review by wzhudev — APPROVED** [review]
LGTM. Thanks!

**Review by wzhudev — APPROVED** [review]
LGTM. Thanks!

**Comment by wzhudev** [comment]
I will fix the CI issue in another PR.

**Pull request merged**